src/core/indexer/mod.rs

```diff
@@ -632,6 +632,11 @@ pub(crate) trait Indexer: Default {
 
         Ok(())
     }
+
+    fn raw_size(&self) -> Result<u32> {
+        let sizes = tables::sizage(&self.code())?;
+        Ok((sizes.fs - (sizes.hs + sizes.ss)) * 3 / 4)
+    }
```

Comment:
https://github.com/WebOfTrust/keripy/blob/development/src/keri/core/coring.py#L3699-L3705

This one looks better